src/Sarif.Converters/ToolFormat.cs

```diff
@@ -14,8 +14,10 @@ public enum ToolFormat
         ClangAnalyzer,
         /// <summary>CppCheck's file format.</summary>
         CppCheck,
-        /// <summary>Fortify's file format.</summary>
+        /// <summary>Fortify's report file format.</summary>
         Fortify,
+        /// <summary>Fortify's FPR file format.</summary>
+        FortifyFpr,
```

Comment:
Yes, this would be a breaking change to utility users but we don't have a large user base for this app. :)

---
In reply to: [94533957](https://github.com/Microsoft/sarif-sdk/pull/554#discussion_r94533957) [](ancestors = 94533957)